lib/paper_trail/record_trail.rb

```diff
@@ -474,5 +489,41 @@ def version
     def versions
       @record.public_send(@record.class.versions_association_name)
     end
+
+    def attribute_in_previous_version(attr_name)
+      if @in_after_callback && rails_51?
+        @record.attribute_before_last_save(attr_name.to_s)
+      else
+        @record.attribute_was(attr_name.to_s)
+      end
+    end
+
+    def changed_in_latest_version
+      if @in_after_callback && rails_51?
+        @record.saved_changes.keys
+      else
+        @record.changed
+      end
+    end
+
+    def changes_in_latest_version
+      if @in_after_callback && rails_51?
+        @record.saved_changes
+      else
+        @record.changes
+      end
+    end
+
+    def attribute_changed_in_latest_version?(attr_name)
+      if @in_after_callback && rails_51?
+        @record.saved_change_to_attribute?(attr_name.to_s)
+      else
+        @record.attribute_changed?(attr_name.to_s)
+      end
+    end
+
+    def rails_51?
+      ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 1
```

Comment:
This can also be written as `ActiveRecord.version >= Gem::Version.new("5.1.0.alpha")` if we don't need to support Rails 3. Unsure which you'd prefer, so I opted for the non-allocating version since this seems like it's called from a semi-hotpath